css-canvas-medical/styles.css

```diff
@@ -0,0 +1,220 @@
+.body {
```

Comment:
I would rename this class to something like `container`, I initially thought you were applying this styling to the `body` element